setup.php

```diff
@@ -75,39 +43,81 @@
         $configured = false;
     }
 
-	$file_written = false;
+?>
 
-    if(is_writeable($configfile) && !$configured){
+<?php if($configured): //TODO, should this reveal exactly how to reconfigure? I don't this it would be a security issue ?>
```

Comment:
Telling people to remove config.php if they want to re-run the set-up should not give away any secret information, I think.